MetasysServices/Exceptions/MetasysException.cs

```diff
@@ -0,0 +1,39 @@
+using System;
+using Flurl;
+using Flurl.Http;
+using System.Web;
+
+namespace JohnsonControls.Metasys.BasicServices
+{
+    [System.Serializable]
+    public class MetasysHttpException : FlurlHttpException {
+        public MetasysHttpException(HttpCall call, string message, Exception inner) : this(call, message, null, inner) { }
```

Comment:
It would be good practice to define a base class for all Metasys exceptions so that a client could catch all Metasys exceptions with one catch handler (if they found it desirable)

Reply:
Resolved in another PR: #37